src/Component/bookList.js

```diff
@@ -1,44 +1,22 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './book';
 
 const BooksList = () => {
-  const books = [
-    {
-      id: 1,
-      name: 'Ada the pretty girl',
-      category: 'Sci-Fi',
-      author: 'Adaobi Okoye',
-    },
-    {
-      id: 2,
-      name: 'Ada the pretty girl',
-      category: 'Sci-Fi',
-      author: 'Adaobi Okoye',
-    },
-    {
-      id: 3,
-      name: 'Ada the pretty girl',
-      category: 'Sci-Fi',
-      author: 'Adaobi Okoye',
-    },
-    {
-      id: 4,
-      name: 'Ada the pretty girl',
-      category: 'Sci-Fi',
-      author: 'Adaobi Okoye',
-    },
-  ];
+  const books = useSelector((state) => state.books);
 
   return (
     <div>
       <ul className="listBooks">
```

Comment:
- [ ] When I add a book, it doesn't show. Your initial state which you populated with some books doesn't show either. This is because you are returning `state.books` instead of `state.book`. In your reducer, your book reducer is assigned to the key `book` and not `books`. Kindly implement this change. 👍